### PR TITLE
Adapting the material section

### PR DESCRIPTION
**Pull request by agude**
Bringing over some of the old material section.

In the other section we sort of talk about rolling but I guess that waits for later. I didn't bring over the character sheet section, trying to keep it light and I think that belongs where we talk about character creation?

### CODE REVIEW

File: _chapters/00-outline.md [hunk 1]
```diff
@@ -21,8 +23,15 @@ The core mechanic of {GameName} is Aspects: short improvised phrases about the i
 
 ## Game Materials
 
-poker chips, dice, paper.
-keep it concise
+{GameName} doesn't require a lot to get started: just pens to write with, paper to record your character on, a set of four Fate dice, and some tokens to represent _Fate Points_.
```
Comment (charles-uno) [inline _chapters/00-outline.md:26]: Do we have a consistent rule about when we're using italics?

File: _chapters/00-outline.md [hunk 2]
```diff
@@ -21,8 +23,15 @@ The core mechanic of {GameName} is Aspects: short improvised phrases about the i
...
+
+Fate dice (sometimes called Fudge dice) are six-sided dice marked with {{pos}} on two sides, {{zero}} on two sides, and {{neg}} on two sides.
+To keep things moving quickly, it's best to have a set of four dice for each player rather than passing them back and forth.
```
Comment (charles-uno) [inline _chapters/00-outline.md:29]:
```suggestion
To keep things moving quickly, it's best to have a set of four dice for each player.
```

File: _chapters/00-outline.md [hunk 3]
```diff
@@ -21,8 +23,15 @@ The core mechanic of {GameName} is Aspects: short improvised phrases about the i
...
+If you don't have specially marked Fate dice, you can use a normal six-side die and treat 1 and 2 as <span class="nowrap">{{neg}},</span> 3 and 4 as <span class="nowrap">{{zero}},</span> and 5 and 6 as <span class="nowrap">{{pos}}.</span>
```
Comment (charles-uno) [inline _chapters/00-outline.md:30]: Might be nice opportunity to break up the wall of text with a list or table

| D6 Roll | Fate Roll |
|:--------:|:--------:|
| 1 or 2 | <span class="nowrap">{{neg}}</span> |
| 3 or 4 | <span class="nowrap">{{zero}}</span> |
| 5 or 6 | <span class="nowrap">{{pos}}</span> |

File: _chapters/00-outline.md [hunk 4]
```diff
@@ -21,8 +23,15 @@ The core mechanic of {GameName} is Aspects: short improvised phrases about the i
...
+
+You'll also need a way to keep track of fate points.
+The best way is to have something physical to pass back and forth---poker chips are excellent for this---but you can track them by just making a mark on your character sheet as well.
```
Comment (charles-uno) [inline _chapters/00-outline.md:33]: We just used "best" in the previous paragraph. Maybe "it's nice to have something physical"?

File: _chapters/00-outline.md [hunk 5]
```diff
@@ -21,8 +23,15 @@ The core mechanic of {GameName} is Aspects: short improvised phrases about the i
...
+In a pinch, you could even use polyhedral dice since you won't otherwise need them in {GameName}.
```
Comment (charles-uno) [inline _chapters/00-outline.md:34]:
```suggestion
You'll also need a way to keep track of fate points. 
Poker chips are perfect for this, but in a pinch you can just mark them down with pencil and paper.
```

Too terse?

**Review by charles-uno — APPROVED** [review]

File: _chapters/00-outline.md (hunk 1 above)
Comment (agude) [reply _chapters/00-outline.md:26]: No... I think it used to be the "first use of a 'reserved keyword'" as it were. :-/ I believe we were trying solve the forward declaration problem by marking these words as special.

Can we do without it? Or instead make it every instance of certain words? I think I prefer without.

File: _chapters/00-outline.md (hunk 3 above)
Comment (agude) [reply _chapters/00-outline.md:30]: Love it.

File: _chapters/00-outline.md (hunk 1 above)
Comment (agude) [reply _chapters/00-outline.md]: @charles-uno I'm good to merge this but thought we might want to come to an agreement on this.

In general how do we discuss stuff like this and save it? Issues tickets?

Comment (agude) [reply _chapters/00-outline.md]: Going to remove italics for now, easy to add back if we want them.

Comment (charles-uno) [reply _chapters/00-outline.md]: My (weak) preference is no italics. Easy enough to later add that back, or weird caps, or whatever

I think discussion in PR is fine for this stuff. Or maybe chat. Turns I don't check my email that often anymore now that it's mostly robots

**Pull request merged**